src/pages/index.tsx

```diff
@@ -1,12 +1,29 @@
+import { GetServerSideProps } from 'next';
+import App from '../App';
+import { parseEmbedQuery } from '../utils';
+
 /**
  * This is a shim for now. Our app currently doesn't use
  * next routing - we're just loading the entire app in _app.tsx
  *
  * This will be fixed in a later PR, adding routes with server
  * rendering to grab OG images.
  */
-const HomePage = () => {
-  return null;
+const HomePage = ({ query }: { query: any }) => {
+  const embed = parseEmbedQuery(query);
+  return (
+    <App
+      embed={{
+        ...embed,
+        isEmbedded: false,
+        embedUrl: null,
+      }}
```

Comment:
This is a little bit strange to me - I don't see how `embed` prop is relevant for a non-embed route. This makes me think that this is really not an embed configuration but rather a set of feature flags, available for any route. Which is fine - but then I would definitely not call this `embed` because it makes the thing confusing

Reply:
it's fixed in the last changes.